doc/source/notebooks/basics/monitoring.pct.py

```diff
@@ -8,8 +8,9 @@
 #       format_version: '1.3'
 #       jupytext_version: 1.4.0
 #   kernelspec:
-#     display_name: 'Python 3.7.4 64-bit (''gpflow2'': conda)'
-#     name: python37464bitgpflow2conda97d7f93b14fc49f496551bb0d0f2e18e
```

Comment:
When reviewing new notebooks we'll have to double check that the kernelspec is set correctly to `Python 3`. This kernelspec was added automatically bu `jupytext` and is custom to my machine and conda env, and thus not available on circle ci.